config/config.msft.clouds-overlay.yaml

```diff
@@ -149,7 +149,7 @@ clouds:
                   sha: 5c3fd99a70fb43aa5ea4672ccaf04c68ac60d7adb7e205d2bf75d1b46395455c
               prometheusSpec:
                 image:
-                  sha: 645b6390daa8fc99985cf40c6868aee92f3ea36b6e92fdf121348e3b26288923
+                  sha: 70296b2f2cc69c88d070fe8b627d347bd070e084c14babbc93e91583e03eca06
```

Comment:
We need to bump in aks first - https://github.com/Azure/ARO-HCP/blob/main/config/config.yaml#L351